frontend/store/companies/mutations.js

```diff
@@ -0,0 +1,29 @@
+import types from "./mutation-types";
```

Comment:
I would call this just `mutation`. They you will call `mutation.SET_COMPANIES` for example.

Reply:
I think mutation-types is more descriptive

Reply:
I meant `types` name, not file name.

Reply:
Okay, already changed.